src/bp/nlu/stan/model-repo.ts

```diff
@@ -1,31 +1,90 @@
 import crypto from 'crypto'
 import fse, { WriteStream } from 'fs-extra'
 import _ from 'lodash'
-import mkdirp from 'mkdirp'
 import * as NLUEngine from 'nlu/engine'
 import path from 'path'
+import Logger from 'simple-logger'
 import { Stream } from 'stream'
 import tar from 'tar'
 import tmp from 'tmp'
+import {
+  Database,
+  DBStorageDriver,
+  DiskStorageDriver,
+  GhostService,
+  ScopedGhostService,
+  MemoryObjectCache
+} from './simple-ghost'
+interface FSDriver {
+  driver: 'fs'
+}
+
+interface DBDriver {
+  driver: 'db'
+  dbURL: string
+}
+
+export type ModelRepoOptions = (FSDriver | DBDriver) & {
+  modelDir: string
+}
+
+interface ModelOwnershipOptions {
+  appId?: string
+  appSecret?: string
+}
 
-export default class ModelRepository {
-  constructor(private modelDir: string) {}
+const MODELS_DIR = './models'
+const debug = DEBUG('nlu').sub('model-repo')
+const defaultOtpions: ModelRepoOptions = {
+  modelDir: path.join(process.cwd(), 'botpress-nlu'),
```

Comment:
We previously used `process.APP_DATA_PATH`. What's your preference for default directory ?

Reply:
Once we extract bp nlu in it's own bin process.appdata will be undefined(stan won't use the bp bootstrap scripts) except if we set it . I set it to make it work but if we don't forget it, then process.appdata it is.